### PR TITLE
add documentation how to access pull-through proxy stats

### PR DESCRIPTION
**Pull request by xoxys**
Adds minimal documentation how to access pull-through cache stats exposed by `expvar` as discussed in https://github.com/distribution/distribution/issues/3407

### CODE REVIEW

File: docs/configuration.md
Comment (milosgajdos) [inline docs/configuration.md:924]: This should be NOTE not NONE, I presume.

Comment (xoxys) [reply docs/configuration.md]: Thanks, fixed.

**Review by milosgajdos — APPROVED** [review]
LGTM. Would be good to get these into `2.7` branch.

**Comment by xoxys** [comment]
Anything required from my side to get it into 2.7 branch? Or is the backport done after merge by one of you?

**Comment by milosgajdos** [comment]
Feel free to open a PR on 2.7 branch. Otherwise one of the maintainers will have to pick it up when free.

**Review by joaodrp — APPROVED** [review]

**Pull request merged**
